cypress/support/fragments/inventory/InventoryInstanceEdit.js

```diff
@@ -81,12 +81,15 @@ export default {
   },
 
   addIdentifier:(identifier) => {
-    cy.do([
-      Button('Add identifier').click(),
-      Accordion('Identifier').find(Select({ name:'identifiers[0].identifierTypeId' })).choose(identifier),
-      TextField({ name:'identifiers[0].value' }).fillIn(value),
-      saveAndCloseButton.click()]);
+    cy.expect(Accordion('Identifier').exists());
+    cy.do(Button('Add identifier').click());
+    cy.expect(Select('Type*').exists());
+    cy.expect(TextField('Identifier*').exists());
+    cy.do(Accordion('Identifier').find(Select({ name:'identifiers[0].identifierTypeId' })).choose(identifier));
```

Comment:
I suggest Accordion('Identifier') and other repeating elements move to const

Reply:
Done